qiita_client/qiita_client.py

```diff
@@ -166,16 +166,16 @@ class QiitaClient(object):
         The client id to connect to the Qiita server
     client_secret : str
         The client secret id to connect to the Qiita server
-    server_cert : str, optional
-        The server certificate, in case that it is not verified
+    ca_cert : str, optional
+        CA cert used to sign and verify cert@server_url
```

Comment:
Is this going to change things in other plugins?

Reply:
only if and when they're supplying the parameter in their test code. Both parameters are identical functionally, just renamed to be more appropriate.